presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -1474,6 +1474,9 @@ public void testColumnAliases()
         assertQuery(
                 "SELECT x, T.y, z + 1 FROM (SELECT custkey, orderstatus, totalprice FROM orders) T (x, y, z)",
                 "SELECT custkey, orderstatus, totalprice + 1 FROM orders");
+
+        // wildcard from aliased table with column aliases
+        assertQuery("SELECT a, b, c FROM (SELECT T.* FROM region T (a, b, c))");
```

Comment:
Shouldn't we have a separate test class for these tests. I don't think there is a point to run this test for every connector. 
Are there any more tests like that besides these two queries?

This is just a thought that should probably be handled as separate PR.

@kasiafi Would you like to handle that or should I log it as github issue?

Reply:
The issue with aliases applies to several connectors (Kafka, Cassandra,...). I think this is the right place to add a test.

Reply:
This is related to the discussion we had on Slack the other day, where I mentioned we need to add a way to create engine SQL tests that operate on synthetic tables. Then we could synthesize one with hidden columns just for that case.

Reply:
Do we want to wait with this fix until we prepare a better place to test it?

Reply:
No, let's keep this test here. No point in forcing it to waiting behind those improvements.